olm-catalog/serverless-operator/index/configs/index.yaml

```diff
@@ -3,22 +3,34 @@ schema: olm.channel
 name: stable
 package: serverless-operator
 entries:
-  - name: "serverless-operator.v1.27.0"
   - name: "serverless-operator.v1.28.0"
-    replaces: "serverless-operator.v1.27.0"
-    skipRange: ">=1.27.0 <1.28.0"
   - name: "serverless-operator.v1.29.0"
     replaces: "serverless-operator.v1.28.0"
     skipRange: ">=1.28.0 <1.29.0"
+  - name: "serverless-operator.v1.30.0"
+    replaces: "serverless-operator.v1.29.0"
+    skipRange: ">=1.29.0 <1.30.0"
 ---
 schema: olm.channel
 name: stable-1.29
 package: serverless-operator
 entries:
-  - name: "serverless-operator.v1.27.0"
   - name: "serverless-operator.v1.28.0"
-    replaces: "serverless-operator.v1.27.0"
-    skipRange: ">=1.27.0 <1.28.0"
   - name: "serverless-operator.v1.29.0"
     replaces: "serverless-operator.v1.28.0"
     skipRange: ">=1.28.0 <1.29.0"
+  - name: "serverless-operator.v1.30.0"
+    replaces: "serverless-operator.v1.29.0"
+    skipRange: ">=1.29.0 <1.30.0"
```

Comment:
We will need to add a bit more [sophisticated generator](https://github.com/openshift-knative/serverless-operator/blob/main/hack/generate/index.sh#L26-L39) for the OLM channels. The 1.29 a previous version need to stop at 1.29. 

```
- name: "serverless-operator.v1.29.0"
    replaces: "serverless-operator.v1.28.0"
    skipRange: ">=1.28.0 <1.29.0"
```

Reply:
/cc @Kaustubh-pande